### PR TITLE
Add zero commit to create repo by default

### LINKED ISSUE #1417: Commit zero when repository created
lakeFS by default should create an empty commit when repository is created.

### PR DESCRIPTION
**Pull request by itaiad200**
close #1417

### CODE REVIEW

File: graveler/ref/manager.go [hunk 1]
```diff
@@ -41,7 +42,17 @@ func (m *Manager) GetRepository(ctx context.Context, repositoryID graveler.Repos
 	return repository.(*graveler.Repository), nil
 }
 
-func (m *Manager) CreateRepository(ctx context.Context, repositoryID graveler.RepositoryID, repository graveler.Repository, branch graveler.Branch) error {
+const (
+	firstCommitMsg = "lakeFS repository created!"
```
Comment (nopcoder) [inline graveler/ref/manager.go:46]: 1. Public under graveler so we can verify when testing.
2. Consider "Initial repository created" or something without lakeFS.

File: graveler/ref/manager.go [hunk 2]
```diff
@@ -41,7 +42,17 @@ func (m *Manager) GetRepository(ctx context.Context, repositoryID graveler.Repos
...
+)
+
+func (m *Manager) CreateRepository(ctx context.Context, repositoryID graveler.RepositoryID, repository graveler.Repository, token graveler.StagingToken) error {
+	firstCommit := graveler.Commit{
+		Message:      firstCommitMsg,
+		CreationDate: time.Now(),
```
Comment (nopcoder) [inline graveler/ref/manager.go:52]: verity if you need to call UTC or the pg lib does that for you

File: graveler/ref/manager.go (hunk 1 above)
Comment (itaiad200) [reply graveler/ref/manager.go:46]: Done. "Initial" for repository is kinda weird, so kept it simple.

File: graveler/ref/manager.go (hunk 2 above)
Comment (itaiad200) [reply graveler/ref/manager.go:52]: It is done by the `addCommit` method below.

**Review by nopcoder — APPROVED** [review]

**Pull request merged**